File: test-suite/src/test/java/io/micronaut/docs/ioc/introspection/JavaxTransientTest.java
package io.micronaut.docs.ioc.introspection;

import io.micronaut.core.beans.BeanIntrospection;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertTrue;

class JavaxTransientTest {
    @Test
    void testIntrospectionWithJavaxTransient() {
        BeanIntrospection<ObjectWithJavaxTransient> introspection = BeanIntrospection.getIntrospection(ObjectWithJavaxTransient.class);
        assertTrue(introspection.getProperty("tmp").isPresent());
    }
}

File: test-suite-kotlin2-ksp2/src/test/kotlin/io/micronaut/docs/ioc/introspection/JavaxTransientTest.kt
package io.micronaut.docs.ioc.introspection

import io.micronaut.core.beans.BeanIntrospection
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions;

class JavaxTransientTest {
    @Test
    fun testIntrospectionWithJavaxTransient() {
        val introspection: BeanIntrospection<ObjectWithJavaxTransient> = BeanIntrospection.getIntrospection(ObjectWithJavaxTransient::class.java);
        Assertions.assertTrue(introspection.getProperty("tmp").isPresent())
    }
}
